skribenten-web/frontend/src/routes/saksnummer_/$saksId/brevvelger/$templateId/-components/BrevmalBrevbaker.tsx

```diff
@@ -0,0 +1,118 @@
+import { css } from "@emotion/react";
+import { ArrowRightIcon } from "@navikt/aksel-icons";
+import { Button, Select, VStack } from "@navikt/ds-react";
+import { useQuery } from "@tanstack/react-query";
+import { useNavigate } from "@tanstack/react-router";
+import { useMemo } from "react";
+import { Controller, useForm } from "react-hook-form";
+import type { z } from "zod";
+
+import { getEnheter } from "~/api/skribenten-api-endpoints";
+import { ApiError } from "~/components/ApiError";
+import { Divider } from "~/components/Divider";
+import type { LetterMetadata } from "~/types/apiTypes";
+import { SpraakKode } from "~/types/apiTypes";
+import { SPRAAK_ENUM_TO_TEXT } from "~/types/nameMappings";
+
+import { Route } from "../route";
+import { PersonAdresse } from "./Adresse";
+import LetterTemplateHeading from "./LetterTemplate";
+import type { brevmalBrevbakerFormSchema } from "./TemplateUtils";
+
+const BrevmalBrevbaker = (properties: { letterTemplate: LetterMetadata; preferredLanguage: SpraakKode | null }) => {
+  const { saksId } = Route.useParams();
+  const navigate = useNavigate({ from: Route.fullPath });
+  const enheterQuery = useQuery(getEnheter);
+  const sorterteSpråk = useMemo(() => {
+    return properties.letterTemplate.spraak.toSorted();
+  }, [properties.letterTemplate.spraak]);
+
+  const form = useForm<z.infer<typeof brevmalBrevbakerFormSchema>>({
+    defaultValues: {
+      spraak:
+        properties.preferredLanguage && sorterteSpråk.includes(properties.preferredLanguage)
+          ? properties.preferredLanguage
+          : sorterteSpråk[0],
+      enhetsId: "",
+    },
+  });
+
```

Comment:
For doksys har vi en useEffect som resetter mutation+form. Hva er grunnen til at vi ikke trenger det her?

Reply:
Vil det si at vi egentlig ikke trenger det i Doksys da?